### PR TITLE
USHIFT-3482: IPv6 handling adjustments in configuration

### PR DESCRIPTION
**Pull request by pacevedom**
**Which issue(s) this PR addresses**:

Closes #<Issue Number>

### CODE REVIEW

File: pkg/util/net.go [hunk 1]
```diff
@@ -117,15 +116,6 @@ func RetryTCPConnection(ctx context.Context, host string, port string) bool {
 	return status
 }
 
-func CreateLocalhostListenerOnPort(port int) (tcpnet.Listener, error) {
```
Comment (pmtk) [inline pkg/util/net.go:120]: I think etcd needs to re-vendor microshift

Comment (pacevedom) [reply pkg/util/net.go:120]: Done

**Pull request merged**